### DISCUSSION TITLE
What does premium theme mean

### DISCUSSION THREAD

**Discussion by rouke-broersma** [Lednerb/bilberry-hugo-theme#311] [Q&A] [answered]
It is unclear to me what the premium classifier is supposed to mean. Am I supposed to pay to use this theme? If so, how? Or does it simply mean that you think it's a high quality theme (from first looks I would agree mind you)? It's kinda confusing to me. I assume that it means the latter since it's MIT licensed.

**Comment by igor-baiborodine** [comment, marked as answer]
You can use the Bilberry theme free of charge. Here `premium` means `superior in quality` and `feature-rich`.
